entities/verbosity/verbosity.go

```diff
@@ -14,6 +14,7 @@ package verbosity
 import "fmt"
 
 const (
+	OutputBatch   = "batchStatus"
 	OutputMinimal = "minimal"
```

Comment:
what do you think maybe instead of adding a new output, we would have a dedicated endpoint just for getting the batchStatus?
```
GET /v1/nodes/batch-status
```